app/internal/domain/service/slack_response_test.go

```diff
@@ -62,6 +62,7 @@ func Test_slackResponseService_postEmailList(t *testing.T) {
 		emails []*model.SlackUserEmail
 		cid    string
 		ts     string
+		user   string
```

Comment:
[IMO] I think the variable name of `userID` is better than `user` because the argument indicating this value of `postEmailList` is `userID`.

Reply:
Thank you. I'll fix it and merge this PR